src/App.vue

```diff
@@ -15,5 +15,6 @@ export default {
 </script>
 
 <style lang="scss">
-  // import global scss
+// import global scss
+@import '../src/styles/style.scss';
```

Comment:
@FlorinMDX ceva nu e ok cu pathul aici '../src/styles/style.scss';  , App e direct in src nu vad de ce te duci cu un nivel mai jos si apoi iarasi src ./styles/style.scss e suficient

Reply:
@ionutvladu am modificat pathu-ul, am inlocuit pro cu free si am schimbat si icon-urile care erau pe versiunea pro